libs/zgui/src/gui.zig

```diff
@@ -3371,6 +3371,88 @@ extern fn zguiGetMouseDragDelta(button: MouseButton, lock_threshold: f32, delta:
 extern fn zguiResetMouseDragDelta(button: MouseButton) void;
 //--------------------------------------------------------------------------------------------------
 //
+// Drag and Drop
+//
+//--------------------------------------------------------------------------------------------------
+pub const DragDropFlags = packed struct(u32) {
+    source_no_preview_tooltip: bool = false,
+    source_no_disable_hover: bool = false,
+    source_no_hold_open_to_others: bool = false,
+    source_allow_null_id: bool = false,
+    source_extern: bool = false,
+    source_auto_expire_payload: bool = false,
+
+    _padding0: u4 = 0,
+
+    accept_before_delivery: bool = false,
+    accept_no_draw_default_rect: bool = false,
+    accept_no_preview_tooltip: bool = false,
+    accept_peek_only: bool = false,
+
+    _padding1: u18 = 0,
+};
+
+const Payload = extern struct {
+    data: *anyopaque = null,
+    data_size: i32 = 0,
+    source_id: u32 = 0,
+    source_parent_id: u32 = 0,
+    data_frame_count: i32 = -1,
+    data_type: [32 + 1]u8,
+    preview: bool = false,
+    delivery: bool = false,
+
+    /// `pub fn init() Payload`
+    pub const init = zguiImGuiPayload_Init;
```

Comment:
We should careful here. User's might expect `zgui.Payload.init()` to behave the same way as the C++ constructor.

I think we should call `Clear` here to mirror the C++ API i.e.
```zig
pub fn init() Payload {
    var payload = Payload{};
    payload.clear();
    return payload;
}
```